### PR TITLE
Website: Add docs for new controls + update screenshots

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

New controls:
iOS: Tooltip
Android: ListItem, Drawer, Circular Progress

Screen shot updates:
Both: Date and time ranges
iOS: List cells, Pivot
Android: People Picker

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9425)

### CODE REVIEW

**Comment by aneeshack4** [comment]
Just putting the do not merge label on for now so no PRs get merged while we're locked down finishing up fabric 7.

File: apps/fabric-website/src/pages/Controls/SpinnerPage/docs/android/SpinnerOverview.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+When you've exhausted all technical avenues for making a user flow faster, a progress indicator should be used to give user feedback that the system is busy. Use these feedback animations when you expect the user will have to wait for 2 seconds or more.
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/SpinnerPage/docs/android/SpinnerOverview.md]: Maybe?
```suggestion
When you've exhausted all technical avenues for making a user flow faster, a progress indicator should be used to give the user feedback that the system is busy. Use these feedback animations when you expect the user will have to wait for 2 seconds or more.
```

File: apps/fabric-website/src/pages/Controls/SpinnerPage/docs/android/SpinnerOverview.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+For pauses between 2 sec and 400ms, adding animations may actually make the experience feel more chaotic. Under 400ms, most delays in loading new content are perceived as fast, and no feedback is needed.
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/SpinnerPage/docs/android/SpinnerOverview.md]: Probably use of abbreviations (or not) should be consistent

File: apps/fabric-website/src/pages/Controls/TooltipPage/docs/ios/TooltipOverview.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Use tooltips to show small unobtrusive hints on top of your app's UI. These can appear as a result of user interaction, or triggered thoughtfully to assist the user in learning the details of a new feature they have shown interest in.
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/TooltipPage/docs/ios/TooltipOverview.md]:
```suggestion
Use tooltips to show small unobtrusive hints on top of your app's UI. These can appear as a result of user interaction, or be triggered thoughtfully to assist the user in learning the details of a new feature they have shown interest in.
```

**Pull request merged**